### ISSUE TITLE
Unable to reproduce SMD

### ISSUE THREAD

**Issue by juntaozhang** [Jhryu30/AnomalyBERT#4]
Thanks for your wonderful work!

As shown in the figure below, I tried many times but couldn't reproduce SMD, was it my command or some other problem。

`python3 train.py --dataset=SMD --default_options=SMD`

![image](https://github.com/Jhryu30/AnomalyBERT/assets/7509285/0402b4dd-fbb9-479a-9e8c-b4fb347cf23b)

![image](https://github.com/Jhryu30/AnomalyBERT/assets/7509285/12bf3c04-9ebe-482f-9900-bf2d35da50fc)

**Comment by coffeetumbler** [comment]
Hello, @juntaozhang.

Have you trained a single network using the entire SMD dataset? If you pre-processed the raw data with our code, you may obtain SMD sub-datasets (SMD0~SMD27) and the aggregated dataset (SMD). We trained 28 models on all of these sub-datasets and the averaged evaluation scores, which are presented in the table. Unfortunately, it is hard to achieve good performance with the aggregated dataset.

If you have any questions or other issues, please let us know. Thank you.

**Comment by juntaozhang** [comment]
Thank you for your reply, I See, I use aggregated dataset (SMD) for test, let me try again seperatly use sub-datasets
